test/integration/consul-container/libs/service/connect.go

```diff
@@ -167,29 +167,24 @@ func NewConnectService(ctx context.Context, sidecarCfg SidecarConfig, serviceBin
 	namePrefix := fmt.Sprintf("%s-service-connect-%s", node.GetDatacenter(), sidecarCfg.Name)
 	containerName := utils.RandName(namePrefix)
 
-	envoyVersion := getEnvoyVersion()
 	agentConfig := node.GetConfig()
-	buildargs := map[string]*string{
-		"ENVOY_VERSION": utils.StringToPointer(envoyVersion),
-		"CONSUL_IMAGE":  utils.StringToPointer(agentConfig.DockerImage()),
-	}
-
-	dockerfileCtx, err := getDevContainerDockerfile()
-	if err != nil {
-		return nil, err
-	}
-	dockerfileCtx.BuildArgs = buildargs
-
 	internalAdminPort, err := node.ClaimAdminPort()
 	if err != nil {
 		return nil, err
 	}
 
+	fmt.Println("agent image name", agentConfig.DockerImage())
+	imageVersion := ""
+	if strings.Contains(agentConfig.DockerImage(), "local") {
+		imageVersion = "target-version"
+	} else {
+		imageVersion = "latest-version"
+	}
```

Comment:
Not blocking: maybe having this logic close to `utils/version.go` would keep more context regarding the image name logic? Applies to all 3 files.

Reply:
Sounds good. Updated.